### PR TITLE
chore(deps): [release-1.3] bump go-toolset container image

### PR DESCRIPTION
**Pull request by coreydaley**
## Which issue(s) does this PR fix or relate to

- Fixes #_issue_number_

## PR acceptance criteria

- [ ] Tests
- [ ] Documentation
- [ ] If the bundle manifests have been updated, make sure to review the [`rhdh-operator.clusterserviceversion.yaml`](../.rhdh/bundle/manifests/rhdh-operator.clusterserviceversion.yaml) file accordingly

### CODE REVIEW

**Review by kim-tsao — COMMENTED** [review]
/lgtm

File: docker/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 #@follow_tag(registry.redhat.io/rhel9/go-toolset:latest)
 # https://registry.access.redhat.com/ubi9/go-toolset
-FROM registry.access.redhat.com/ubi9/go-toolset:1.21.13-2.1729155367@sha256:b08eed026ac732d125e61ebfe3a0ef1dea7293b6a1c6d8b71c007a3088c4ed3b AS builder
+FROM registry.access.redhat.com/ubi9/go-toolset:1.21.13-2.1727893526@sha256:b03f78d00ca116ffc0e025028ee063305d0f92a1cfcff3ba9e14e4a680cdcb35 AS builder
```
Comment (nickboldt) [inline docker/Dockerfile:17]: https://github.com/redhat-developer/rhdh-operator/blob/release-1.3/.rhdh/docker/Dockerfile#L6 also needs updating.

Comment (coreydaley) [reply docker/Dockerfile:17]: done.

**Review by nickboldt — CHANGES_REQUESTED** [review]
This is updating from new timestap 1729... to older timestamp 1727... 

and it's the direct opposite of this:

https://github.com/redhat-developer/rhdh-operator/pull/351/files

**Pull request closed without merging**